kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
shouldn't this be protected, as only a single level is stored, so multiple calls fail? eg. insert "assert(not old_lvl, "log already disabled")